ansible/roles/software/datagov-deploy-fgdc2iso/tasks/main.yml

```diff
@@ -4,21 +4,32 @@
     msg: saxon_license is required but has not been defined.
   when: saxon_license is undefined
 
-- name: include distribution-specific variables
-  include_vars: "{{ ansible_distribution_release }}.yml"
-
-- name: prepare packages needed for harvester
-  apt: name="{{ catalog_harvester_fgdc2iso_apt_packages }}" state=present
```

Comment:
In terms of removing the existing tomcat package, what are you thinking? Should removing the package be a one-off as part of the next release?

Reply:
Yes, one-time ansible command to remove them after deployment.